templates/data/data.html

```diff
@@ -1,52 +1,75 @@
 {%- extends 'base.html' -%}
 {%- block title -%}API - AcousticBrainz{%- endblock -%}
 {%- block content -%}
-   <h3>AcousticBrainz API</h3>
-   <p>
-      Before diving into the API details, please have a look at the
-      <a href="#sample-data">sample data</a> below.
-   </p>
-   <p>
-      To fetch the JSON documents for a given MusicBrainz recording ID,
-      you can construct the following URLs:
-   </p>
+  <h3>AcousticBrainz API</h3>
+  <p>
+    Before diving into the API details, please have a look at the
+    <a href="#sample-data">sample data</a> below.
+  </p>
+
+  <h4>Getting data</h4>
+  <p>
+    To fetch the JSON documents for a given MusicBrainz recording ID, you can
+    construct the following endpoints for low-level and high-level:
+  </p>
 <pre>
    GET http://acousticbrainz.org/&lt;MBID&gt;/low-level
    GET http://acousticbrainz.org/&lt;MBID&gt;/high-level
 </pre>
-   <p>Examples:</p>
+  <p><em>Examples:</em></p>
 <pre>
    GET <a href="http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/low-level">http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/low-level</a>
    GET <a href="http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/high-level">http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/high-level</a>
 </pre>
-<p>The submit API end-point is:</p>
+  <p>
+    You can find out how many low-level documents we have for a given track
+    using the following endpoint:
+  </p>
+<pre>
+   GET http://acousticbrainz.org/&lt;MBID&gt;/count
+</pre>
+  <p><em>Example:</em></p>
+<pre>
+   GET <a href="http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/count">http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/count</a>
+</pre>
+  <p>If there's more than one document, you can specify a number argument <em>n</em>:</p>
+<pre>
+   GET http://acousticbrainz.org/&lt;MBID&gt;/low-level?n=1
+</pre>
+  <p><em>Example:</em></p>
+<pre>
+   GET <a href="http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/low-level?n=1">http://acousticbrainz.org/96685213-a25c-4678-9a13-abd9ec81cf35/low-level?n=1</a>
+</pre>
```

Comment:
"documents are counted from 0 to count-1, and matches the suffixes added to documents in the json download"